manage.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+
+from argparse import ArgumentParser, Namespace, _SubParsersAction
+from os import path as P
+from pathlib import PurePath
+import subprocess
+from typing import Callable
+
+LANGKIT_ROOT = PurePath(P.dirname(P.realpath(__file__)))
+LKT_LIB_ROOT = LANGKIT_ROOT / "contrib" / "lkt"
+PYTHON_LIB_ROOT = LANGKIT_ROOT / "contrib" / "python"
+
+
+def create_subparser(
+    subparsers: _SubParsersAction,
+    fn: Callable[[Namespace], None]
+) -> ArgumentParser:
+    """
+    Create a subparser with given ``fn`` as func. Extract doc and name from
+    the function.
+    """
+    subparser = subparsers.add_parser(name=fn.__name__, help=fn.__doc__)
+    subparser.set_defaults(func=fn)
+    return subparser
+
+
+def setenv(args: Namespace) -> None:
+    """
+    Setenv for langkit.
+    """
+    for cwd in (LKT_LIB_ROOT, PYTHON_LIB_ROOT):
+        subprocess.check_call(["./manage.py", "setenv"], cwd=cwd)
+
+
+def make(args: Namespace) -> None:
+    """
+    Make langkit.
+    """
```

Comment:
Please also port the removal of the build directories. We can get mysterious dynamic link errors without them.